docs/SRS-Volere/SRS.tex

```diff
@@ -49,7 +51,8 @@ \section*{Revision History}
 \begin{tabularx}{\textwidth}{p{3cm}p{3cm}X}
   \toprule {\textbf{Date}} & {\textbf{Author}} & {\textbf{Notes}}\\
   \midrule
-  October 11 2024 & Entire Team & Rev0 of the SRS\\
+  10/11/2024 & Entire Team & Rev0 of the SRS\\
+  11/04/2024 & Rafeed Iqbal & Updated sections 13 and 15\\
```

Comment:
write it as two entries (because two unrelated things were done), and mention the id's of the requirements removed and added, so
```suggestion
  11/04/2024 & Rafeed Iqbal & Removed OE-IAS1-2\\
  11/04/2024 & Rafeed Iqbal & Added SR-S1\\
```